example/main.go

```diff
@@ -139,12 +139,14 @@ func resolveTOTPSecretReference(client *onepassword.Client, vaultID, itemID, fie
 }
 
 func createAndGetItem(client *onepassword.Client) onepassword.Item {
+	vaultID := os.Getenv("OP_VAULT_ID")
+
```

Comment:
Thanks for this enhancement Morgan! 

Can we also add in `example/README.md` the prerequisite to setting this as an envvar?

We want to make sure that the users that will be running this don't have to surface this implementation detail themselves.